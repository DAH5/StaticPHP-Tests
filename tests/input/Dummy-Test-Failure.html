<p>Dummy test failure.</p>