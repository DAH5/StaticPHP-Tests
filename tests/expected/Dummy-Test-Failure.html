<p>Dummy test failure. It should fail.</p>